core/src/banking_stage.rs

```diff
@@ -960,13 +961,32 @@ impl BankingStage {
         chunk_offset: usize,
         transaction_status_sender: Option<TransactionStatusSender>,
         gossip_vote_sender: &ReplayVoteSender,
+        cost_model: &Arc<RwLock<CostModel>>,
     ) -> (Result<usize, PohRecorderError>, Vec<usize>) {
-        let mut lock_time = Measure::start("lock_time");
+        let mut qos_service_stats = QosServiceStats::default();
+        let qos_service = QosService::new(cost_model.clone());
+        let tx_costs = qos_service.compute_transactions_costs(
+            txs.iter(),
+            bank.demote_program_write_locks(),
+            &mut qos_service_stats,
+        );
+
+        let transactions_status = qos_service.select_transactions_per_cost(
```

Comment:
Maybe name this variable something a little more descriptive? Perhaps `transaction_qos_results`? I think there's precedent for using "XXX_results" elsewhere.

Reply:
Good call, `xxx_results` is more consistent. Let's do that